bin/pipeline.sh

```diff
@@ -25,3 +28,16 @@ pipetask run -j "$1" -b "$2"/butler.yaml \
     --input "$INPUTCOLL" --output "$COLLECTION" \
     --register-dataset-types \
     --qgraph "$QGRAPH_FILE"
+
```

Comment:
Can I just suggest copy/pasting the #TODO comment above this line? Just so we can make sure the exclusion is removed in the fakes run as well whenever DM-29751 gets solved.